src/components/toolbox/sliderCheckbox/index.js

```diff
@@ -117,45 +109,38 @@ class SliderCheckbox extends React.Component {
         onMouseUp={this.stopTracking.bind(this)}
         onTouchEnd={this.stopTracking.bind(this)}>
         <span
-          onClick= {this.change.bind(this)}
           className={`${theme.circle} ${theme.button} circle`}
           ref={(el) => { this.shape = el; }}>
           <span className={theme.arrowRight}>
-            { textAsIcon ?
-              <span className={theme.text}>{icons.unchecked}</span> :
-              <FontIcon className={theme.icon}>
-                {icons.unchecked || 'arrow-right'}
-              </FontIcon>
-            }
+            <FontIcon className={theme.icon}>
+              {icons.unchecked || 'arrow-right'}
+            </FontIcon>
           </span>
 
           <span className={theme.checkMark}>
-            { textAsIcon ?
-              <span className={theme.text}>{icons.checked}</span> :
-              <FontIcon className={theme.icon}>
-                {icons.checked || 'checkmark'}
-              </FontIcon>
-            }
+            <FontIcon className={theme.icon}>
+              {icons.checked || 'checkmark'}
+            </FontIcon>
           </span>
         </span>
         { label ?
           <div>
-            <span>{label}</span>
+            <span className='label'>{label}</span>
```

Comment:
I would suggest to use less generic class names for unit tests, there might be class collisions coming from 3rd party libs? 
Maybe better to scope it to the component name? `slider-check-label` , `slider-check-arrow`

Reply:
We don't use 3rd party libraries unless we isolate them with. css modules. this is the beauty of our CSS stack which allows us use more human readable names.